data/en/4.0.affix

```diff
@@ -39,13 +39,17 @@ _ ‐ ‑ ‒ – — ― ～ – ━ ー -- - ‧
 -- ‒ – — ― "(" ")" "[" "]" ... ";" ±: MPUNC+;
 
 % Suffixes
-'s 're 've 'd 'll 'm ’s ’re ’ve ’d ’ll ’m: SUF+;
+'s 're 've 'em 'd 'll 'm 'd've 'r 't
+’s ’re ’ve ’em ’d ’ll ’m ’d’ve ’r ’t: SUF+;
```

Comment:
Duplication here.

Reply:
they're not the same mark; one is vertical, one is slanted. ox27 and 0xe28099=U+2019

it would be nice if either could be used, automatically...